Sources/RingPublishingTracking/Private/EventsFactory+Audio.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EventsFactory+Audio.swift
+//  RingPublishingTracking
+//
+//  Created by Bernard Bijoch on 08/10/2024.
+//  Copyright © 2024 Ringier Axel Springer Tech. All rights reserved.
+//
+
+import Foundation
+
+extension EventsFactory {
+
+    func createAudioEvent(audioEvent: AudioEvent, audioMetadata: AudioMetadata, audioState: AudioState) -> Event {
+
+        var parameters: [String: AnyHashable] = [:]
+        parameters["VE"] = audioEvent.rawValue
+        parameters["RT"] = EventType.videoEvent.rawValue
+        parameters["PMU"] = audioMetadata.contentId
+        parameters["VP"] = audioState.currentTime
+        parameters["VC"] = createAudioEventVCParameter(metadata: audioMetadata, audioState: audioState)
+        parameters["RR"] = audioEventSessionTimestamp(for: audioMetadata.contentId, audioEvent: audioEvent)
+        parameters["VEN"] = audioEventSessionCounter(for: audioMetadata.contentId, audioEvent: audioEvent)
+        parameters["VS"] = audioMetadata.audioStreamFormat.rawValue
+        parameters["VSM"] = startMode
+        parameters["XI"] = audioMetadata.audioPlayerVersion
+        parameters["VSLOT"] = playerType
+        parameters["VEM"] = mainAudio
+        parameters["VPC"] = audioMetadata.audioContentCategory.rawValue
+        parameters["ECX"] = createContextParam(visibility: audioState.visibilityState, output: audioState.audioOutput)
+        parameters["FRA"] = audioMetadata.isContentFragment
+        parameters["VT"] = audioMetadata.audioDuration
+
+        if  [AudioEvent.start, .playingStart, .autoPlayingStart].contains(where: { $0 == audioEvent }) {
```

Comment:
spacja

@danielcalka mieliśmy to w Androidzie? bo to w sumie mamy zapisane by zrobić - wyszło by w testach też

Reply:
`         if (audioEvent == AudioEvent.START || audioEvent == AudioEvent.PLAYING_START || audioEvent == AudioEvent.PLAYING_AUTOSTART) {
                this[AudioEventParam.AUDIO.text] = audioMetadata.createAudioJson()
            }`
            
            Jeśli mówisz o tym, to mamy